### DISCUSSION TITLE
fine tuning for first and last names?

### DISCUSSION THREAD

**Discussion by silvacarl2** [openai/whisper#1338] [Q&A]
we want to fine tune on the 12M first and last names we have to get the accuracy up as high as possible.

we would be willing to share this with anyone that might want to help us.

question: is it best to keep the first and last names together as one audio file for fine tuning?  or is it better to break them up into first names audio files and last names audio files for fine tuning?

carl

**Comment by phineas-pta** [comment]
why don't you use `prompt` to add new vocabulary ?

**Comment by silvacarl2** [comment]
we use initial prompt now.  the issue is if they are calling in for the first time, there is no way to know what their name could be.  so it has to just take the recording and do the best it can with it.

which is about 82% accurate.

since we have 12M samples with ground truth, we thoguht we could get that number much higher.

thoughts?

**Comment by dgoryeo** [comment]
@silvacarl2 have you considered [WhisperBiasing](https://github.com/BriansIDP/WhisperBiasing)? I'm not sure if that requires the 12M names to be labeled with corresonding sound in a dataset.

**Comment by silvacarl2** [comment]
NICE!!!!!!!!!!!!!!!!!

p.s. if you would like to assist us on this we can pay for assistance.  8-)

**Reply by dgoryeo** [reply]
Haha -- @silvacarl2  I was actually looking for a small gig, my planned gig was just delayed by 1 month.

**Comment by silvacarl2** [comment]
silvacarl@gmail.com
or
csilva@sphinxmedtech,com

**Comment by silvacarl2** [comment]
Ok, email me if you have time/are interested.

 

We have 14M names to test with.
